kernel/syscall.c

```diff
@@ -1541,6 +1541,29 @@ long myst_syscall_mkdir(const char* pathname, mode_t mode)
     return ret;
 }
 
+long myst_syscall_mkdirat(int dirfd, const char* pathname, mode_t mode)
+{
+    char* abspath = NULL;
+    long ret = 0;
+
+    if (dirfd == AT_FDCWD)
+    {
```

Comment:
I think `myst_get_absolute_path_from_dirfd` covers the case of `AT_FDCWD`

Reply:
That's right, it does. The redundant check is now removed.